src/cli.ts

```diff
@@ -55,15 +56,15 @@ const cli = meow(
   }
 )
 
-export const options = ({
-  [FLAGS.COMMIT]: (options: Object) => commands.commit(options),
+export const options = {
+  [FLAGS.COMMIT]: (options: any) => commands.commit(options),
```

Comment:
These are typed as `any` for now until we transition the commands to typescript and the correct `Options` type can be exported.